NOTICE.txt

```diff
@@ -9,6 +9,15 @@ Third party libraries used by the Elastic Beats project:
 ==========================================================================
 
 
+--------------------------------------------------------------------
+Dependency: cloud.google.com/go
```

Comment:
Could you check how much the binary size increases with these new dependencies?

Reply:
Master -> 72740520 bytes ~70mb
PR -> 72716960 bytes ~70mb

Reply:
It seems with the PR the build is smaller? Not sure how this is possible. Perhaps new stuff in master and not rebase yet?

Reply:
You're were right, a rebase was needed. But after rebasing the sizes are now equals which doesn't make sense either :confused: 
Master -> 72740520
PR -> 72740520

Reply:
how do you build it? is the import correct? Are you looking at the x-pack one?

Reply:
Actually, I was doing it in x-pack and the sizes were also equal, so I switched to OSS because I thought that there was something I missed from the current build system in X-Pack but the sizes were also equal...  I have had problems in the past where we needed to delete the contents in `$GOPATH/pkg` to solve some library issues but even with this the binary sizes are equal

Reply:
The problem is that `_ github.com/elastic/beats/x-pack/metricbeat/include` is missing in the main.go file so imports are never loaded. That also means probably your testing is not working? I get the two following values:

```
Before: 57200688
After: 58519728
```

Reply:
Good catch @ruflin I think it's not surprising that the tests have passed because there are not system tests in xp/metricbeat yet (which is the one that compiles the binary) and the integration tests (the ones run with `mage goItengTest`) compiles at the package level. When testing in local I was working normally, with iterations of failing tests, fixing, etc.

So, the package itself is 9260 bytes in a single file https://github.com/elastic/beats/pull/9414/files#diff-563909ea0b3926d894389fca8b784bdb plus another 11358 of the ASL2 file.

Go source code is just importing `fmt` and `time` so I'd say that the increase in size comes 99% from the mssql library itself.